.env

```diff
@@ -0,0 +1 @@
+COMPOSE_PROJECT_NAME=hca
```

Comment:
Not sure what your intended nomenclature is here, but this repo is for the HCA DCP DSS (Human Cell Atlas Data Coordination Platform Data Storage System) so calling this "hca" could be overly general.